### PR TITLE
OSDOCS-4989: Tech Preview - Admin Network Policy and Baseline Admin Network Policy

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-4989

Link to docs preview:
https://65682--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-enabling-features

QE approval by asood-rh

### CODE REVIEW

**Review by tssurya — APPROVED** [review]
small suggestion for a new line, otherwise lgtm
thanks @ahardin-rh !

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 1]
```diff
@@ -31,6 +31,7 @@ The following Technology Preview features are enabled by this feature set:
...
 ** Pod security admission enforcement. Enables the restricted enforcement mode for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards. (`OpenShiftPodSecurityAdmission`)
 ** StatefulSet pod availability upgrading limits. Enables users to define the maximum number of statefulset pods unavailable during updates which reduces application downtime. (`MaxUnavailableStatefulSet`)
+** Admin Network Policy and Baseline Admin Network Policy. Enables `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy API, in the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and guardrails for an entire cluster before namespaces are are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary.
```
[the remark this answers is not in the document]
Comment (tssurya) [reply modules/nodes-cluster-enabling-features-about.adoc:34]: oh and I also noted that the rest of the points have the feature gate name in the brackets for end of each bullet. In our case we want the `AdminNetworkPolicy` in that special font... in brackets -> that is the feature name to enable both these features.

**Review by tssurya — APPROVED** [review]
/lgtm

**Comment by ahardin-rh** [comment]
@memodi @weliang1 Can you please provide QE review?

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 2]
```diff
@@ -31,6 +31,7 @@ The following Technology Preview features are enabled by this feature set:
...
 ** Pod security admission enforcement. Enables the restricted enforcement mode for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards. (`OpenShiftPodSecurityAdmission`)
 ** StatefulSet pod availability upgrading limits. Enables users to define the maximum number of statefulset pods unavailable during updates which reduces application downtime. (`MaxUnavailableStatefulSet`)
+** Admin Network Policy and Baseline Admin Network Policy. Enables `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. These APIs currently support only expressing policies for intra-cluster traffic. (`AdminNetworkPolicy`)
```
Comment (skopacz1) [inline modules/nodes-cluster-enabling-features-about.adoc:34]: One small suggestion to space out the two adverbs for somewhat easier reading, no big deal either way:

```suggestion
** Admin Network Policy and Baseline Admin Network Policy. Enables `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently these APIs support only expressing policies for intra-cluster traffic. (`AdminNetworkPolicy`)
```

**Review by skopacz1 — COMMENTED** [review]
Just left one suggestion, otherwise looks good!

**Comment by tssurya** [comment]
@asood-rh has also being reviewing this feature, she might be able to review this

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 3]
```diff
@@ -31,6 +31,7 @@ The following Technology Preview features are enabled by this feature set:
...
 ** Pod security admission enforcement. Enables the restricted enforcement mode for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards. (`OpenShiftPodSecurityAdmission`)
 ** StatefulSet pod availability upgrading limits. Enables users to define the maximum number of statefulset pods unavailable during updates which reduces application downtime. (`MaxUnavailableStatefulSet`)
+** Admin Network Policy and Baseline Admin Network Policy. Enables `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic. (`AdminNetworkPolicy`)
```
Comment (asood-rh) [inline modules/nodes-cluster-enabling-features-about.adoc:34]: @ahardin-rh Should it be "resources, which are part of the Network Policy V2 API" referring to calling out  **V2 API** here?

Comment (ahardin-rh) [reply modules/nodes-cluster-enabling-features-about.adoc:34]: @asood-rh I'm sorry. I don't understand your comment. Can you please clarify? Thanks!

Comment (tssurya) [reply modules/nodes-cluster-enabling-features-about.adoc:34]: I think the sentence is ok, this is just a snippet to let users know what's the new thing about. 
"resources, which are part of the Network Policy V2 API," is what is written right?

Comment (ahardin-rh) [reply modules/nodes-cluster-enabling-features-about.adoc:34]: correct; thanks!

Comment (tssurya) [reply modules/nodes-cluster-enabling-features-about.adoc:34]: @asood-rh 's confusion was around the whole "V2" part which makes sense - reading https://network-policy-api.sigs.k8s.io/ will help.

- NetworkPolicy in tree object is referred to as V1
- ANP, BANP (future DNP) are referred to as V2 of Network Policy APIs fondly.

**Pull request merged**